Firestore/core/test/unit/api/aggregate_query_test.cc

```diff
@@ -37,16 +38,31 @@ class MockAggregateQuery : public AggregateQuery {
  public:
   StatusOr<ObjectValue> mockResult;
 
+  ~MockAggregateQuery() override = default;
```

Comment:
You don't need to re-declare the destructor in subclasses. You can just omit it if its body is empty.

Reply:
The c++ style guide says to annotate overrides of virtual functions and destructors with "override" or "final". See Classes > Inheritance. Since the default destructor is an override of the parent virtual destructor, then depending on interpretation, it needs to be annotated with "override" in this re-declaration. Thoughts?

Reply:
I just removed this so it doesn't prolong this PR.